src/mod.rs

```diff
@@ -0,0 +1,13 @@
+pub mod error;
```

Comment:
I think this file is not required. everything's in lib already?

Reply:
Compiles without mod.rs file deleting